packages/vue/src/components/organisms/SfFooter/SfFooter.stories.js

```diff
@@ -63,34 +78,69 @@ const Template = (args, { argTypes }) => ({
           title: "Payment & delivery",
           items: ["Purchase terms", "Guarantee"],
         },
-        {
-          title: "Social",
-          pictures: ["facebook", "pinterest", "twitter", "youtube"],
-        },
       ],
+      social: {
+        title: "Social",
+        pictures: ["Facebook", "Pinterest", "Twitter", "Google", "Youtube"],
+      },
     };
   },
   template: `
     <SfFooter
       :column="column"
       :multiple="multiple"
       :open="open"
+      :logo="logo"
+      :title="title"
       class="sb-footer"
     >
       <SfFooterColumn v-for="column in columns" :key="column.title" :title="column.title">        
-        <SfList v-if="column.items">
+        <SfList>
           <SfListItem v-for="item in column.items" :key="item">
             <SfMenuItem :label="item"/>
           </SfListItem>
-        </SfList>
-        <div v-else class="sb-footer__social">
-          <SfImage v-for="picture in column.pictures" :key="picture" width="12" height="12" :src="'/assets/storybook/SfFooter/'+picture+'.svg'" :alt="picture" class="sb-social-icon"/>
-        </div>
+        </SfList>      
+      </SfFooterColumn>
+      <SfFooterColumn title="social.title" class="desktop-only">
+        <SfImage v-for="picture in social.pictures" :key="picture" :src="'/assets/storybook/SfFooter/'+picture+' Icon.svg'" :alt="picture" class="sb-social-icon"/>
+      </SfFooterColumn> 
+      <SfFooterColumn title="Language" class="desktop-only">
+        <SfButton class="sb-footer__button sf-button--pure">
+          <SfImage
+            src="/assets/storybook/SfFooter/flag.png"
+            alt="flag of the USA"
+          />
+          English
+        </SfButton>
+      </SfFooterColumn>
+      <SfFooterColumn class="desktop-only">
+        <template>
+          <div class="sb-footer__input">
```

Comment:
this class has too big width which causes on mobile view a scrollbar on bottom :)

Reply:
additinal scss file is removed so this will be also fixed